specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/stable/2019-03-01/AlertsManagement.json

```diff
@@ -52,6 +52,42 @@
         }
       }
     },
+    "/providers/Microsoft.AlertsManagement/alertsMetaData": {
```

Comment:
The response defined in this spec does not conform to the RPC.  This is a collection GET and should follow the "paging response body" schema here:  https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/resource-api-reference.md#get-resource